### PR TITLE
Make the question better match quest_dietType_explanation

### PR DESCRIPTION
**Pull request by peternewman**
For example olives would be "any vegetarian/vegan items" but aren't a comparable choice. Any is "not none", which isn't really what the longer description aims at.

### CODE REVIEW

**Comment by westnordost** [comment]
Didn't you notice the

> Only answer “yes” if the place provides a comparable choice of eating options for this diet

?

**Comment by peternewman** [comment]
Yes, I did, and I'm aware of what the question is so go pretty much entirely on the icon, but as has been proved time and again, many people don't read the smaller text, so getting the main question to cover it as effectively as possible without making it too wordy seems very beneficial to me.

**Comment by westnordost** [comment]
It's casual, yes, but that doesn't mean it is not well translatable. As it happens, casual language is usually better translatable to my experience.

**Comment by westnordost** [comment]
Anyway, the current text came from a discussion on OSM-Slack US (#language channel). I learnt that e.g. "veg item on the menu" is synonymous to "veg option" and both mean proper meals, not "an item in a list". I am not convinced that your suggestion would be an improvement in clarity.

**Comment by peternewman** [comment]
> It's casual, yes, but that doesn't mean it is not well translatable. As it happens, casual language is usually better translatable to my experience.

Casual language is generally more ambiguous so more chance of confusion.

> Anyway, the current text came from a discussion on OSM-Slack US (#language channel). I learnt that e.g. "veg item on the menu" is synonymous to "veg option" and both mean proper meals, not "an item in a list".

I'm not sure item has the same significance this side of the pond, compared to the vegetarian option.

> I am not convinced that your suggestion would be an improvement in clarity. **Edit**: IIRC the previous text was "Are there any...", so "Any" is just the casual form of that

The previous text, as per @FloEdelmann excellent diff ( https://github.com/streetcomplete/StreetComplete/pull/3870#discussion_r825452124 ), was:
```diff
@@ AddVegan @@
- Does […] have vegan meals on the menu?
+ Any vegan items on the menu here?

@@ AddVegetarian @@
- Does […] have vegetarian meals on the menu?
+ Any vegetarian items on the menu here?
```

Would you accept changing it to meals (Any veg meals on the menu here?), which seems like a much better fit all round (and matches the more detailed text)?

I suspect the underlying issue is it's trying to fulfil two purposes (and I suspect the reason someone suggest item), if you go to a doughnut shop, a single doughnut isn't a meal, but it is an item, but you're only buying one thing. But as part of a meal, appetisers/starters/sides are far more likely to be veg friendly, but you'll be unimpressed if that's all you get.

**Comment by westnordost** [comment]
>> It's casual, yes, but that doesn't mean it is not well translatable. As it happens, casual language is usually better translatable to my experience.

> Casual language is generally more ambiguous so more chance of confusion.

Well it looks like we have a different opinion here. Note that I wrote "casual", not "colloquial" or "slang". In my experience, the casual language that is spoken on casual/normal interactions is the first thing you learn to use right.

> I'm not sure item has the same significance this side of the pond, compared to the vegetarian option.
> Would you accept changing it to meals (Any veg meals on the menu here?)

 I wouldn't accept changing it to meals because this is the very thing the people in the #language channel found sounded odd. "This is something a waiter would say". But well, if the understanding differs on the other side of the pond, that's good, so then you can use a better fitting wording for en-GB any everybody is happy.

**Comment by mnalis** [comment]
see also https://github.com/streetcomplete/StreetComplete/issues/2539 - "comparable" was key word to me to indicate that beefsteak house which besides beefsteaks has vegetarian soups and salads and potatoes as side dishes does not count as "yes" in that quest.
